src/lib.rs

```diff
@@ -1153,12 +1155,6 @@ mod panic {
 
 struct LoggerGuard(&'static Log);
 
-impl Drop for LoggerGuard {
```

Comment:
This is why I left it here after I refactored my first pass, which misunderstood the task. But on account of my initial confusion, I can certainly imagine that I'm missing something.